providers/common/der/build.info

```diff
@@ -5,10 +5,12 @@ $DER_DIGESTS_H=$INCDIR/der_digests.h
 $DER_DIGESTS_GEN=der_digests_gen.c
 
 GENERATE[$DER_DIGESTS_GEN]=der_digests_gen.c.in
+DEPEND[GENFIPSFILES]=$DER_DIGESTS_GEN
```

Comment:
Instead of doing this would it rather make sense to just go through the generated files in the build tree as well? Just list all generated files in the build tree matching *.c and *.h? This would avoid the *.in -> * hack in the script.

Reply:
The advantage of using `git ls-files` is that if there is other random test programs or other detritus in a working tree it will be ignored. I think that's more common.

Reply:
Not here it isn't more common.....

Reply:
You mean not more common in the directory?  I meant overall in the source tree which is why I like `git ls-files`

Shrug.  Happy to close this PR or have someone else take it over.